schema.sql

```diff
@@ -0,0 +1,10 @@
+/* Database schema to keep the structure of entire database. */
+
+CREATE TABLE animals (
+  id INT GENERATED BY DEFAULT AS IDENTITY,
```

Comment:
- [ OPTIONAL] I would highly recommend using the PRIMARY KEY (id) in your database design. :wink: The primary key is a crucial element in a database table as it uniquely identifies each record in the table. By defining the id column as the primary key, you ensure that each row in the table has a unique identifier. This helps in maintaining data integrity and ensures that there are no duplicate records.

Reply:
Thank you @Alfredbis29.